taichi/llvm/llvm_context.cpp

```diff
@@ -38,12 +38,20 @@
 #include "taichi/jit/jit_session.h"
 #include "taichi/common/task.h"
 #include "taichi/util/environ_config.h"
-#ifdef _WIN32
-// Travis CI seems doesn't support <filesystem>...
-#include <filesystem>
+
+#ifdef TI_PLATFORM_OSX
+#include "Availability.h"
+#ifndef __MAC_10_15
+// Travis CI is not yet on Mac 10.15...
+#define _TI_USE_EXPERIMENTAL_FILESYSTEM 1
+#endif  // __MAC_10_15
+#endif  // TI_PLATFORM_OSX
+
+#ifdef _TI_USE_EXPERIMENTAL_FILESYSTEM
```

Comment:
Can't we just always use `<experimental/filesystem>`? It works on new machines too.

Reply:
Thanks. <experimental/filesystem> on my local env generated warnings, and it didn't fix the problem anyway. I'm now trying to just roll up XCode in Travis CI and see if that fixes the error. Will let you know once this is ready.